README.md

```diff
@@ -1 +1,189 @@
-# capstone-ebike-reservation
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h1><b>Ebike Reservation BackEnd</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+## 📗 Table of Contents
```

Comment:
- [ ]  The "test" folder, which is used by a test framework that is not being used. Kindly consider deleting that folder if you are not using it. 👍

- [ ] OPTIONAL: Include some tests.